wireapp/wire-android-reloaded
# feat: classified domains calling context (AR-1557)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1557" title="AR-1557" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />AR-1557</a>  Federated status of remote call participant
  </td></table>
  <br />
 










----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Creates a component to display conversation/call classification banner and applied into calling context, incoming and ongoing calls.

#### How to Test

You can create an account into `ANTA` or `BELLA` and do some calls between groups or one to one call across backends.
You should see the banner as `Classified` accordingly if they are supported as feature flag and participants are trusted.

### Notes (Optional)

More context about classification of conversations and backends can be found here:

https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/376439791/Use+case+determine+if+a+conversation+or+user+is+classified+non-classified

### Attachments (Optional)

<img src="https://user-images.githubusercontent.com/5806454/187691430-075638cd-cebc-47f2-9cba-b73c9abfcc98.png" width="300"/>
<img src="https://user-images.githubusercontent.com/5806454/187691443-26f24f6f-5711-4aa4-ad83-bef65a9c75b2.png" width="300"/>
<img src="https://user-images.githubusercontent.com/5806454/187691459-7de23d3e-1022-468b-a3aa-f6ab54b3b7ed.png" width="300"/>

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
